### ISSUE TITLE
Saving camera settings results in NatsError: TIMEOUT

### ISSUE THREAD

**Issue by leigh-johnson** [bitsy-ai/printnanny-os#266]
## **Describe the bug**
Saving camera settings using the form results in NatsError: TIMEOUT

**Comment by leigh-johnson** [comment]
Looks like we're not deserializing caps correctly:

```
Feb 15 21:29:05 travelpi printnanny-edge-nats[682]: [2023-02-15T22:29:05Z ERROR printnanny_settings::cam] Failed to parse i32 from caps width=Err(ValueGetError { name: "width", error: ValueTypeMismatchError { actual: GstIntRange, requested: gint } }) with error=GetError: Structure field with name width not retrieved
29:05 travelpi printnanny-edge-nats[682]: [2023-02-15T22:29:05Z ERROR printnanny_settings::cam] Failed to get colorimetry caps, using default: bt701 error=GetError: Structure field with name colorimetry not found
Feb 15 21:29:05 travelpi printnanny-edge-nats[682]: [2023-02-15T22:29:05Z ERROR printnanny_settings::cam] Failed to parse i32 from caps height=Err(ValueGetError { name: "height", error: ValueTypeMismatchError { actual: GstIntRange, requested: gint } }) with error=GetError: Structure field with name height not retrieved
Feb 15 21:29:05 travelpi printnanny-edge-nats[682]: [2023-02-15T22:29:05Z ERROR printnanny_settings::cam] Failed to parse i32 from caps width=Err(ValueGetError { name: "width", error: ValueTypeMismatchError { actual: GstIntRange, requested: gint } }) with error=GetError: Structure field with name width not retrieved
Feb 15 21:29:07 travelpi printnanny-edge-nats[682]: [2023-02-15T22:29:07Z ERROR printnanny_nats::subscriber] Error deserializing NATS message: missing field `framerate_d` at line 1 column 210
```

**Comment by leigh-johnson** [comment]
🎉 Verified this is fixed in the latest build. 

<img width="1512" alt="Screenshot 2023-02-16 at 12 24 16 AM" src="https://user-images.githubusercontent.com/2601819/219215314-e7bd8d5e-2dda-4629-8522-44fb364d4971.png">

**Comment by leigh-johnson** [comment]
👍 Confirming this regression is fixed in the latest v0.6.6 nightly
